packages/release_notes/android_bridge.md

```diff
@@ -0,0 +1,14 @@
+# Android Bridge Release Notes
```

Comment:
By using semantic releases, we can have automatically generated release notes

And move our current rigor checks from these files to valid commit/PR titles and bodies

Reply:
I'll use the github releases instead as we discussed live.